docs/landing-page.asciidoc

```diff
@@ -281,4 +282,19 @@ <h4 class="mt-3">
   </div>
 </div>
 
-<p class="my-4"><a href="https://www.elastic.co/guide/index.html">View all Elastic docs</a></p>
+<script>
```

Comment:
This script was copied from the Observability docs:
https://github.com/elastic/observability-docs/blob/2c5ec189bd08f43366a39796137ad5c26c139247/docs/en/observability/landing-page/temp/script.asciidoc?plain=1#L2